include/librealsense2/h/rs_types.h

```diff
@@ -20,6 +20,7 @@ typedef enum rs2_notification_category{
     RS2_NOTIFICATION_CATEGORY_HARDWARE_ERROR,   /**< Error reported from the device */
     RS2_NOTIFICATION_CATEGORY_HARDWARE_EVENT,   /**< General Hardeware notification that is not an error */
     RS2_NOTIFICATION_CATEGORY_UNKNOWN_ERROR,    /**< Received unknown error from the device */
+    RS2_NOTIFICATION_CATEGORY_FIRMWARE_UPDATE_REQUIRED,  /**< Current firmqare version installed is not the latest available **/
```

Comment:
Can the wording be changed to "Recommended" instead of "Required"? 
For consideration